instance/tests/models/test_openedx_instance.py

```diff
@@ -213,6 +213,9 @@ def test_spawn_appserver(self, mocks, privacy_policy_url, mock_consul, mock_prov
         instance = OpenEdXInstanceFactory(
             sub_domain='test.spawn',
             privacy_policy_url=privacy_policy_url,
+            s3_access_key='test',
+            s3_secret_access_key='test',
+            s3_bucket_name='test-s3-bucket-name',
```

Comment:
I would like to avoid having to pass the s3_* fields in explicitly like this. Do tests fail if we remove them? If they're required for tests to work, maybe it would be better to set them in the mocked `provision_s3` method (which is [what would happen](https://github.com/open-craft/opencraft/blob/4b050bfba879dc9c26b2d1490079590374ad4169/instance/models/mixins/storage.py#L297-L298) if the method wasn't mocked out).

Reply:
They are needed, otherwise the `get_storage_settings()` doesn't return the proper output. Which in result causes some errors.

Reply:
I removed them, I'll see what error I get and fix it accordingly, without adding those explicit `s3_*` fields.